File: CodigoFonte/PrefeituraCachoeiro/PrefeituraCachoeiro.Aplicacao/Dtos/Respostas/ProjetoResponse.cs
using System.Diagnostics.CodeAnalysis;

namespace PrefeituraCachoeiro.Aplicacao.Dtos.Respostas
{
    [ExcludeFromCodeCoverage]
    public class ProjetoResponse
    {
        public int IdProjeto { get; set; }
        public string? NomeProjeto { get; set; }
        public int? CodigoProjeto { get; set; }
        public int? IdPrefeitura { get; set; }
        public List<ContratosProjetoDto> Contratos { get; set; }
    }
}

File: Frontend/PrefeituraCachoeiro.App/src/app/response/projetoResponse/projetoResponse.ts
import { ContratoModel, ContratosResponse } from "../contratosResponse/todosContratosResponse";

export class ProjetoResponse{
    idProjeto: number = 0;
    nomeProjeto: string = '';
    nomeContrato?: string;
    idPrefeitura?: number = 0; 
    nomePrefeitura?: string;
    codigoProjeto: number;
    contratos: ContratoModel[];
    acoes?: string;
}

export class MensagemProjetoResponse{
    mensagem: string = '';
}


export class ProjetosResponse {
    data: ProjetoResponse[];
    totalRegisters: number;
  }

  export class RetornaProjetoIdResponse{
    idProjeto: number = 0;
}
